app-admin/logmon/logmon-0.4.4-r1.ebuild

```
@@ -9,7 +9,7 @@ MY_P="LogMon-${PV}"
 DESCRIPTION="Split-screen terminal/ncurses based log viewer"
 HOMEPAGE="http://www.edespot.com/logmon/"
 SRC_URI="http://www.edespot.com/code/LogMon/${MY_P}.tar.bz2"
```

since the manifest changed, you must rename the result (otherwise you will still fetch the old "wrong" tarball from mirrors).

```suggestion
SRC_URI="http://www.edespot.com/code/LogMon/${MY_P}.tar.bz2 -> ${P}.r1.tar.bz2"
```

---

ahh, yes.. updated now